### PR TITLE
fix(seg): maintain algorithm name and algorithm type when DICOM seg is exported or downloaded

### PR DESCRIPTION
**Pull request by IbrahimCSAE**
### Context

This will maintain the AlgorithmName and AlgorithmType when DICOM SEG is exported, it also makes the modality selector searchable, and fixes the study date selector height to match the rest in the study list.

Fixes the https://github.com/OHIF/Viewers/issues/4021

Exported SEG: 

image

### CODE REVIEW

File: platform/ui/src/components/InputMultiSelect/InputMultiSelect.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const InputMultiSelect = ({
         value={value}
         isMulti={true}
         isClearable={false}
-        isSearchable={false}
+        isSearchable={true}
```
Comment (sedghi) [inline platform/ui/src/components/InputMultiSelect/InputMultiSelect.tsx:33]: I think our UI for having searchable is aweful, do you have screenshot to show before and after this change? we should move to the comboBox in the shadcn

**Review by sedghi — COMMENTED** [review]
see my comment

File: platform/ui/src/components/InputMultiSelect/InputMultiSelect.tsx (hunk 1 above)
Comment (IbrahimCSAE) [reply platform/ui/src/components/InputMultiSelect/InputMultiSelect.tsx:33]: It doesn’t look bad to me, I formatted it well, I will post screenshots tomorrow

**Review by sedghi — APPROVED** [review]
Thanks

**Pull request merged**